ftw/file/resources/uploader.js

```diff
@@ -58,9 +58,11 @@ var uploader = (function($) {
       unbindEvents();
       if (tests.dnd) {
         $(document).on('dragenter', function(event) {
-          dragging++;
-          overlay.overlay().load();
-          event.preventDefault();
+          if(event.dataTransfer.types[0] === 'Files') {
```

Comment:
I just tested with images and it worked fine. According to this doc it works for all file types. https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer#types.28.29